src/pages/en/install/auto.md

```diff
@@ -36,62 +37,44 @@ yarn create astro
-- `Blog`, `Documentation`, `Portfolio`: opinionated themes for specific use-cases.
-- `Completely empty`: A template that just includes the bare minimium to get started.
-
-Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.
-
-:::tip[Online previews]
-Want to check out the templates in your browser before choosing? Visit [astro.new](https://astro.new/)
-:::
-
-### Install Dependencies (optional)
-The wizard will offer to run an `install` command for you at this time, which is optional.
+1. Where would you like to create your new Astro project?
+    
+    Astro must be installed in an empty folder. Type in a name for your new project directory.
+    
+    ```bash
+    ./my-astro-site
+    ```
 
-:::caution
-If you do not do so at this time, you will need to [install dependencies](/en/install/auto/#2-install-dependencies) after the wizard has finished, before starting your project.
-:::
-
-### Install any Official Astro Integrations (optional)
-You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components/) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Partytown, Sitemap) by running `astro add --yes`
-
-
-To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle between selected states. You can select multiple items at once, or you can continue without selecting any integrations.
-
-
-When you are satisfied with your selection, press return (enter) to submit.
-
-:::note
-These integrations, and any [Astro community integrations](https://astro.build/integrations/), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide/).
-:::
+1. Which template would you like to use? (browse all templates on [astro.new](https://astro.new/)).
+    
+    Select from the following list:
+    - [Just the Basics](https://github.com/withastro/astro/tree/main/examples/basics) - includes a typical project structure as a model
+    - [Blog](https://github.com/withastro/astro/tree/main/examples/blog) - includes an index page and a folder with first blog post
+    - [Docs](https://github.com/withastro/astro/tree/main/examples/docs) - includes sidebar navigation, "On this page" contents and more!
+    - [Portfolio](https://github.com/withastro/astro/tree/main/examples/portfolio) - styled and ready to go portfolio and about pages!
+    - [Completely Empty](https://github.com/withastro/astro/tree/main/examples/minimal) - just a single page
 
 
-After selecting your integrations to add, you should see the following terminal message notifying you of the changes that `create-astro` will apply to your project's `astro.config.mjs`:
+1. Would you like to run npm install? (Y/n)
```

Comment:
```suggestion
1. Would you like to run npm (yarn, pnpm) install? (Y/n)
```
This message will differ based on the package manager

Reply:
Would you mind if this deviated slightly from the CLI output?

Could say

```suggestion
1. Would you like to install dependencies? (Y/n)
```

Reply:
Or even
```suggestion
1. Would you like to install dependencies with your package manager? (Y/n)
```

Reply:
Yup, that'll do me!  💜